### PR TITLE
Reprocessing News Update

### PR DESCRIPTION
**Pull request by JeffLCoughlin**
This is an update on the global uniform reprocessing effort, highlighting campaigns 2, 3, and 13. Comments are welcome.

### CODE REVIEW

**Comment by JeffLCoughlin** [comment]
I'll plan on making this live by CoB today. If anybody has an issue or wants more time to review, please let me know.

File: content/news/Reproc-Update-1-C2-C13-C3.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+Title: K2 Campaigns 2, 3, and 13 Reprocessed
```
Comment (barentsen) [inline content/news/Reproc-Update-1-C2-C13-C3.md]: `Reprocessed` => `reprocessed`  (we have a convention to use lower case in blog titles)

File: content/news/Reproc-Update-1-C2-C13-C3.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+In the two figures below, we show the current status of major and minor new features and improvements as a result of the reprocessing effort. Below those, we highlight a few examples of these improvements and new features from C2, C3, and C13 that are likely to be of interest to users.
+
+<div class="thumbnail" style="width: 90%; display: inline-block;">
+<div class="caption">
+<i>The status of major pipeline improvements for each campaign as of October 1, 2018. Blue check marks are improvements added a result of the global uniform reprocessing effort.</i>
```
Comment (barentsen) [inline content/news/Reproc-Update-1-C2-C13-C3.md]: I recommend explaining what black check marks mean as well.

File: content/news/Reproc-Update-1-C2-C13-C3.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+</div>
+<img style="text-align: center" src="images/news/c2-dynablack-correction.png" width="100%" align="center"  class="img-responsive">
+</div>
+
+<br>
+
+- As part of the reprocessing, we are now producing short-cadence lightcurves for all objects that were observed in short-cadence mode. We note that no work was able to be done to adapt the detrending method used for Kepler short-cadence lightcurves to those from K2, and thus on a whole they are expected to be of varying quality. However, we are producing them with the expectation that at least some objects will be of high quality, and/or their existence will spurn more detailed investigation. Below we show the short-cadence PDC lightcurve for EPIC 206003187, an RR Lyrae-type variable observed in C3. While there are still residual systematics near thruster-firing events, simply removing those points near thruster firing events results in a well-detrended phased light curve with a clearly visible [Blazhko effect](https://en.wikipedia.org/wiki/Blazhko_effect).
```
Comment (barentsen) [inline content/news/Reproc-Update-1-C2-C13-C3.md]: I don't think it's true that "short cadence detrending was not adapted for K2", because presumably the short cadence detrending does benefit from changes made to PDC to make it work on long cadence K2 data?  I'd rephrase the first sentence to avoid down-playing the effort!

**Comment by barentsen** [comment]
Thanks @JeffLCoughlin!  I made a few very small comments, but overall this looks great!

Perhaps you want to end the blog post with a forward-looking paragraph, i.e. describing what users may expect in the coming months?

File: content/news/Reproc-Update-1-C2-C13-C3.md (hunk 2 above)
Comment (JeffLCoughlin) [reply content/news/Reproc-Update-1-C2-C13-C3.md]: Great call, done.
